guides/Experimental.md

```diff
@@ -1,30 +1,46 @@
 # Experimental recommendations
 
+This document contains suggestions for recommendations to address use cases and problems that users have identified as possible additions to the science-on-schema.org recommendations. These are presented for discussion and testing. Suggestions here should be linked to issues in the Science-On-Schema.org issue tracker. 
 
+# Table of contents
+1. [Linking Physical Samples to a Dataset](#LinkingPhysicalSamples)
+2. [Advanced variable value type description](#AdvancedVariableValueType)
 
-## Attaching Physical Samples to a Dataset
+   2.1 [Variables with non-numeric values](#NonnumericValues)
 
-Currently, there isn't a great semantic property for a Dataset to distinguish the related physical samples. However, we can use the [schema:hasPart](https://schema.org/hasPart) property to accomplish this without too much compromise. A [GitHub issue](https://github.com/earthcubearchitecture-project418/p418Vocabulary/issues/16) has been setup to follow this scenario. Here is the best way, so far, to link physical samples to a Dataset:
+   2.2 [Value range is controlled vocabulary](#ControlledVocabulary)
+
+   2.3 [Structured values](#StructuredValues)
+
+   2.4 [Variables that contain references](#References)
+ 
+   2.5 [Array, Gridded or Coverage Data](#ArrayValues)
+ 
+ 
+<div id='LinkingPhysicalSamples'/>
+
+## Linking  Physical Samples to a Dataset
+
+Currently, there isn't a great semantic property for a Dataset to distinguish the related physical samples. However, we can use the [schema:hasPart](https://schema.org/hasPart) property to accomplish this without too much compromise. A [GitHub issue](https://github.com/earthcubearchitecture-project418/p418Vocabulary/issues/16) has been setup to follow this scenario. Here is a suggested approach to link physical samples to a Dataset:
 
 <pre>
 {
   "@context": {
     "@vocab": "https://schema.org/",
-    "gdx": "https://geodex.org/voc/",
-    <strong>"geolink": "http://schema.geolink.org/1.0/base/main#",
-    "igsn": "http://pid.geoscience.gov.au/def/voc/igsn-codelists/",</strong>
+    
+    <strong>"igsn": "http://igsn.org/",</strong>
   },
   "@type": "Dataset",
   ...,
   <strong>"hasPart": [
     {
-      "@type": ["CreativeWork", "geolink:PhysicalSample"],
+      "@type": ["CreativeWork", "http://vocabulary.odm2.org/specimentype/individualSample/"],
       "identifier": {
-        "@id": "https://doi.org/10273/WHO000A53",
+        "@id": "igsn.org/WHO000A53",
```

Comment:
Shouldn't [nodeIds](https://www.w3.org/TR/json-ld/#node-identifiers) be [IRIs](https://datatracker.ietf.org/doc/html/rfc3987#section-2)? This could be changed to `igsn:WHO000A53` since the namespace is defined in the context.